javalanglib/src/main/scala/java/lang/Character.scala

```diff
@@ -28,30 +30,30 @@ class Character(private val value: scala.Char) extends Comparable[Character] {
   protected def toByte: scala.Byte     = value.toByte
   protected def toShort: scala.Short   = value.toShort
   protected def toChar: scala.Char     = value.toChar
-  protected def toInt: scala.Int       = value
+  protected def toInt: Int       = value
```

Comment:
Hum, don't change places in the code that have nothing to do with the commit. (I guess this was a massing search-and-replace.)